pom.xml

```diff
@@ -70,11 +92,15 @@
                     </execution>
                 </executions>
             </plugin>
-
             <plugin>
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-surefire-plugin</artifactId>
-                <version>2.22.2</version>
+                <version>3.0.0-M4</version>
+                <configuration>
+                    <argLine>
+                        ${access.option}
```

Comment:
If you have to add the argline to the surefire plugin, please check here (https://stackoverflow.com/questions/23190107/cannot-use-jacoco-jvm-args-and-surefire-jvm-args-together-in-maven)

this will make the jacoco agent not generate the report file. mabye this make travis-ci pending

Reply:
Okay I'll check this.